### PR TITLE
s/complient/compliant

### PR DESCRIPTION
**Pull request by PhilETaylor**
s/complient/compliant

### CODE REVIEW

File: modules/mod_breadcrumbs/helper.php [hunk 1]
```diff
@@ -68,7 +68,7 @@ public static function getList(&$params)
 	/**
 	 * Set the breadcrumbs separator for the breadcrumbs display.
 	 *
-	 * @param   string  $custom  Custom xhtml complient string to separate the
+	 * @param   string  $custom  Custom xhtml compliant string to separate the
 	 * items of the breadcrumbs
```
Comment (Quy) [inline modules/mod_breadcrumbs/helper.php]: Align this line under `Custom`

Comment (PhilETaylor) [reply modules/mod_breadcrumbs/helper.php]: All I did was fix the typo!

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 0818b8bf4531404f4f403d4cedc7dba072ea5322<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22995">issues.joomla.org/tracker/joomla-cms/22995</a>.</sub>

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 0818b8bf4531404f4f403d4cedc7dba072ea5322<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22995">issues.joomla.org/tracker/joomla-cms/22995</a>.</sub>

**Comment by infograf768** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22995">issues.joomla.org/tracker/joomla-cms/22995</a>.</sub>

**Pull request merged**